src/instruments/src/Clock/Components/ElapsedTime.tsx

```diff
@@ -6,19 +6,22 @@ const getDisplayString = (seconds: number, running: boolean) : string => (second
 
 export const ElapsedTime = () => {
     const [ltsTest] = useSimVar('L:A32NX_OVHD_INTLT_ANN', 'bool', 250);
+    const [dcEssIsPowered] = useSimVar('L:A32NX_ELEC_DC_ESS_BUS_IS_POWERED', 'bool', 250);
     const [elapsedKnobPos] = useInteractionSimVar('L:A32NX_CHRONO_ET_SWITCH_POS', 'number', 'A32NX_CHRONO_ET_POS_CHANGED');
     const [absTime] = useSimVar('E:ABSOLUTE TIME', 'Seconds', 1000);
     const [prevTime, setPrevTime] = useState(absTime);
 
     const [elapsedTime, setElapsedTime] = useState(0);
 
     useEffect(() => {
-        if (elapsedKnobPos === 0) {
-            setElapsedTime(Math.max(elapsedTime + absTime - prevTime, 0));
-        } else if (elapsedKnobPos === 2) {
-            setElapsedTime(0);
+        if (dcEssIsPowered) {
+            if (elapsedKnobPos === 0) {
+                setElapsedTime(Math.max(elapsedTime + absTime - prevTime, 0));
+            } else if (elapsedKnobPos === 2) {
+                setElapsedTime(0);
+            }
+            setPrevTime(absTime);
         }
-        setPrevTime(absTime);
     }, [absTime, elapsedKnobPos]);
```

Comment:
This should also depend on dsEssIsPowered, shouldn't it?
```suggestion
    }, [absTime, elapsedKnobPos, dcEssIsPowered]);
```

Reply:
Missed that one, and yes, useEffect here would depend on dcEssIsPowered